cmd/system-shutdown/system-shutdown.c

```diff
@@ -75,10 +87,12 @@ int main(int argc, char *argv[])
 			die("cannot move writable out of the way");
 		}
 
-		bool ok = umount_all();
-		kmsg("%c was %s to unmount writable cleanly", ok ? '-' : '*',
-		     ok ? "able" : "*NOT* able");
-		sync();		// shouldn't be needed, but just in case
+                if (umount_all()) {
+                        kmsg("- was able to unmount writable cleanly");
+                } else {
+                        kmsg("* was *NOT* able to unmount writable cleanly");
+                        sync(); // we don't know what happened but we're going ahead
```

Comment:
Or the hard way `echo s > /proc/sysrq-trigger`. Though if there's ever a need to touch sysrq then it's probably too late for cleaning up nicely.

Reply:
at this point there is no echo, and there might be no proc :-)